src/logic/operate.js

```diff
@@ -0,0 +1,36 @@
+import Big from 'big-js';
+
+const Operate = (numOne, numTwo, operation) => {
+  Big.DP = 10;
+  Big.RM = 1;
+  Big.NE = -5;
+  Big.PE = 10;
```

Comment:
what is this for ?? (just curious) 😊

Reply:
@shloch  it is for restricting how many numbers can be returned, more detail you can check here - 
http://mikemcl.github.io/big.js/#dp